packages/tex-to-myst/src/quotes.ts

```diff
@@ -0,0 +1,35 @@
+import type { Handler } from './types.js';
+import { getArguments } from './utils.js';
+
+export const QUOTE_HANDLERS: Record<string, Handler> = {
+  env_quote(node, state) {
+    state.closeParagraph();
+    state.openNode('blockquote');
+    state.renderChildren(node);
+    state.closeNode();
+  },
+  env_displayquote(node, state) {
+    state.closeParagraph();
+    state.openNode('blockquote');
+    state.renderChildren(node);
+    state.closeNode();
+  },
+  macro_epigraph(node, state) {
+    const [quote, author] = getArguments(node, 'group');
+    if (author) {
+      state.openNode('container', { kind: 'quote' });
+    }
+    state.openNode('blockquote');
+    state.openParagraph();
```

Comment:
These `openParagraph`/`closeParagraph` calls aren't in the other handlers - maybe unnecessary? I think the behaviour is the same in each case (looking at test cases), so the differences in the code are a little misleading.